AppServicesUsageSamples/README.md

```diff
@@ -0,0 +1,28 @@
+# App Services Usage Samples
+
+Reference app that show cases different design patterns and examples of using the Realm Kotlin SDK with Atlas App Services.
+
+## Samples
+
+### [Property level encryption](apps/property-encryption/README.md)
+
+This demo shows the process of protecting users' sensitive data by employing end-to-end encryption techniques while guaranteeing access from any user's device.
+
+## Demo app structure
+
+The project has been structured in two main folders:
+
+- Demo - Android app containing the different samples. Samples have been separated in different packages.
+- Apps - Atlas App services apps required by each sample.
+
+## Getting started
+
+The demos are indendepent of each other, this means that it is not required to install all the app services app samples to test an individual sample.
+
+To begin, locate the App services app sources that you wish to install. We have conveniently linked them in the Samples list of this document.
+
+Next, follow the steps outlined in the [Atlas documentation](https://www.mongodb.com/docs/atlas/app-services/apps/create/) to setup the apps. These docs will guide you through the process and help troubleshoot any issue you might encounter.
```

Comment:
I think this is a little sparse. At least I had troubles just getting up and running. Maybe we need to elaborate on the steps and show the actual import commands, or at at least point to some elaborate instructions, because simply creating an app and trying to upload the app didn't do the trick for me. Lets discuss offline.